application/src/main/java/org/togetherjava/tjbot/commands/help/HelpThreadActivityUpdater.java

```diff
@@ -30,9 +34,16 @@ public final class HelpThreadActivityUpdater implements Routine {
     private static final Logger logger = LoggerFactory.getLogger(HelpThreadActivityUpdater.class);
     private static final int SCHEDULE_MINUTES = 30;
     private static final int ACTIVITY_DETERMINE_MESSAGE_LIMIT = 11;
+    private static final int PERSIST_DURATION_VALUE = 12;
+    private static final ChronoUnit PERSIST_DURATION_UNIT = ChronoUnit.HOURS;
 
     private final HelpSystemHelper helper;
 
+    public static final Cache<Long, Long> manuallyResetChannelActivityCache = Caffeine.newBuilder()
```

Comment:
the bot isnt restarted "often" (twice per month). but when it happens, all current questions are reset and everyone who used this command that day will be confused and annoyed.

i dont see it as a high priority, since this feature isnt really crucial anyways, but i would find it better to have this based on a database table instead of a memory-cache.

now, if u say "nah, thats too complex for me for now" or similar, i would be fine with merging this PR as-is and doing the database thing as improvement afterwards.

Reply:
I have very little time due to a new project, and I would prefer if we merge the PR and implement the persistence in a database in a separate issue.